proto/transfer/v1/tx.proto

```diff
@@ -41,7 +41,9 @@ message MsgTransfer {
   uint64 timeout_timestamp = 7
       [ (gogoproto.moretags) = "yaml:\"timeout_timestamp\"" ];
 
-  neutron.feerefunder.Fee fee = 8
+  string memo = 8;
```

Comment:
usually reordering of fields is not a good practice

Reply:
Yes, we want to save compatibility with the original transfer message from ibc-go